### PR TITLE
Fix/delivery report application accreditation

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the accreditation logos in the non accredited delivery-reports 

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to hasta
- [ ] do us
- [ ] do cg upload delivery-report -p hugefilly | grep -F "logo"

### Expected test outcome
- [ ] check that  "logo" was not part of the output
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch
- [ ] Inform to Clinical

### CODE REVIEW

File: cg/cli/get.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def get(context: click.Context, identifier: str):
 def sample(context: click.Context, families: bool, flowcells: bool, sample_ids: List[str]):
     """Get information about a sample."""
     for sample_id in sample_ids:
-        LOG.debug("{sample_id}: get info about sample")
+        LOG.debug(f"{sample_id}: get info about sample")
```
Comment (moonso) [inline cg/cli/get.py:44]:
```suggestion
        LOG.debug("%s: get info about sample", sample_id)
```

Comment (moonso) [reply cg/cli/get.py:44]: Lazy logging is always preferred

**Review by moonso — APPROVED** [review]
👍

**Comment by Mropat** [comment]
Good catch! :laughing:

**Comment by Mropat** [comment]
Logo gone !
image

**Comment by Mropat** [comment]
Shows up on accredited application
image

**Pull request merged**